Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -588,6 +588,7 @@ final class StatusTableViewController: LoopChartsTableViewController {
         case enactingBolus
         case bolusing(dose: DoseEntry)
         case cancelingBolus
+        case pumpSuspended(resuming: Bool)
```

Comment:
[nit] Why not have two separate states for this: `pumpSuspended` and `pumpResuming`?

Reply:
So, I'm just re-adding code that already existed and was removed.